src/btree/bt_read.c

```diff
@@ -18,40 +18,37 @@ int
 __wt_las_remove_block(WT_SESSION_IMPL *session,
     WT_CURSOR *cursor, uint32_t btree_id, const uint8_t *addr, size_t addr_size)
 {
-	WT_DECL_ITEM(las_addr);
-	WT_DECL_ITEM(las_key);
+	WT_ITEM las_addr, las_key, las_timestamp;
 	WT_DECL_RET;
 	uint64_t las_counter, las_txnid, remove_cnt;
 	uint32_t las_id;
 	int exact;
 
 	remove_cnt = 0;
 
-	WT_ERR(__wt_scr_alloc(session, 0, &las_addr));
-	WT_ERR(__wt_scr_alloc(session, 0, &las_key));
-
 	/*
 	 * Search for the block's unique prefix and step through all matching
 	 * records, removing them.
 	 */
-	las_addr->data = addr;
-	las_addr->size = addr_size;
-	las_key->size = 0;
-	cursor->set_key(
-	    cursor, btree_id, las_addr, (uint64_t)0, (uint32_t)0, las_key);
+	las_addr.data = addr;
+	las_addr.size = addr_size;
+	las_key.size = 0;
+	las_timestamp.size = 0;
+	cursor->set_key(cursor, btree_id, &las_addr,
+	    (uint64_t)0, (uint32_t)0, &las_timestamp, &las_key);
```

Comment:
To be clear, I don't think there was anything wrong before, I'd just like to avoid heap allocation unless we really need it.